src/resources/js/ui_online.js

```diff
@@ -343,8 +344,32 @@ export function setUpUI() {
     const roomId = generatePushID();
     createRoom(roomId).then(() => {
       printCurrentRoomID(roomId);
+      copyBtn.style.visibility = 'visible';
```

Comment:
In this project, I prefer using `classList.add`/`classList.remove` to manipulating the styles directly. If it is tiresome for you, I'll fix it.

Reply:
Ok. I don't know if I understand, but I'll try.

Reply:
I'm using `hidden` css class for this.

https://github.com/gorisanson/pikachu-volleyball-p2p-online/blob/66cac2ed98c82856d68c88ed90949fb4136fa251/src/resources/style.css#L393

I think maybe `button.hidden` selector should be added.